.github/workflows/release.yml

```
@@ -68,14 +68,13 @@ jobs:
         deno run `
           --allow-run `
           --allow-net `
-          "${{ github.workspace }}/.github/cicd/validate-tag.ts" `
+          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-tag.ts`
           "${{ inputs.release_type }}" `
           "v${{ steps.get-version.outputs.version }}" `
           "${{ env.project-name }}";
 
     - name: Validate Release Notes
-      run: deno run --allow-read "${{ github.workspace }}/.github/cicd/validate-notes.ts" "${{ inputs.release_type }}" "v${{ steps.get-version.outputs.version }}";
-
+      run: deno run --allow-read "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts`
```

Please fix the `deno run` command.  You added the URL in place of the file path but you removed all of the other parts of the command which is the arguments being sent into the script.

> **Note** Refer to the _**Validate Tag**_ step before this step for reference as a guide.

```suggestion
      run: deno run `
        --allow-read `
        "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts `
        "${{ inputs.release_type }}" `
        "v${{ steps.get-version.outputs.version }}";
```